docs/central-backup.rst

```diff
@@ -7,94 +7,43 @@ Having a data backup strategy is a critical part of running a web service like O
-
-You can also use `the API <https://odkcentral.docs.apiary.io/#reference/system-endpoints/direct-backup/using-an-ad-hoc-passphrase>`_ to download a backup without configuring managed backups.
+For more information, please see the `API documentation <https://odkcentral.docs.apiary.io/#reference/system-endpoints/direct-backup>`_.
 
 .. _central-backup-restore:
 
 Restoring a backup
 ------------------
 
-Restoring a backup to a Central instance will entirely replace all of its data with the backup. Please be very sure you are restoring to the right place with the right backup snapshot before proceeding. 
+Restoring a Direct Backup file to a Central instance will entirely replace all of its data with the backup. Please be very sure you are restoring to the right place with the right backup snapshot before proceeding.
 
 .. note::
 
   You cannot restore a backup to an older version of Central. For example, if you create a backup from Central v1.0, you cannot restore it to Central v0.9.
 
-1. The first thing you'll have to do is download your backup from Google Drive, which you can do from the `Google Drive website <https://drive.google.com/>`_. You will find the backups in a folder called ``ODK Backups``. Each file is a single backup snapshot, and each snapshot should be titled ``backup-{date}T{time}Z.zip``.
+.. note::
+
+  Before v2023.1, Central supported scheduled backups to Google Drive. That functionality has been `sunsetted <https://forum.getodk.org/t/backups-to-google-drive-from-central-will-stop-working-after-jan-31st/38895>`_, but you can follow the same steps here to restore a backup file from Google Drive. The first thing you'll have to do is download your backup from Google Drive, which you can do from the `Google Drive website <https://drive.google.com/>`_. You will find the backups in a folder called ``ODK Backups``. Each file is a single backup snapshot, and each snapshot should be titled ``backup-{date}T{time}Z.zip``.
 
-#. Once you have the file on your local computer, you will have to transfer that backup snapshot file to your ODK Central server. If you don't know how to do this, and you used our DigitalOcean installation guide, please see `their instructions <https://www.digitalocean.com/community/tutorials/how-to-use-sftp-to-securely-transfer-files-with-a-remote-server>`_ on how to transfer a file to a Droplet.
+1. Find the backup file on your local computer. You will have to transfer that backup snapshot to your ODK Central server. If you don't know how to do this, and you used our DigitalOcean installation guide, please see `their instructions <https://www.digitalocean.com/community/tutorials/how-to-use-sftp-to-securely-transfer-files-with-a-remote-server>`_ on how to transfer a file to a Droplet.
```

Comment:
Quick RST question: does the first one need to be `1.`, or can it be `#.`? I tried `#.`, and it seemed fine, but I didn't want to diverge from the existing pattern.

Reply:
#. is better but it doesn’t deal well with nested code blocks, notes, or other directives as far as I have been able to figure out. We should use auto numbering where we can!